check-mysql/lib/readonly.go

```diff
@@ -0,0 +1,42 @@
+package checkmysql
+
+import (
+	"os"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/mackerelio/checkers"
+)
+
+type readOnlyOpts struct {
+	mysqlSetting
```

Comment:
It's better to accept options to specify which status to expect, IS readonly or IS NOT readonly.

Reply:
I see now.
I added a flag parameters and committed it. https://github.com/mackerelio/go-check-plugins/pull/205/commits/b25ceaedecc4a5f01102186752bc406c89a3ea5c